couchbase/datadog_checks/couchbase/couchbase.py

```diff
@@ -402,3 +407,65 @@ def extract_seconds_value(self, value):
             unit = 'us'
 
         return float(val) / TO_SECONDS[unit]
+
+    def _collect_index_stats_metrics(self):
+        url = '{}{}'.format(self._index_stats_url, INDEX_STATS_METRICS_PATH)
```

Comment:
let's use urljoin instead of manual format. It's error prone if a user ends the index url with `/`

```
from six.moves.urllib.parse import urljoin
```